http-protocol-binding.md

```diff
@@ -147,6 +147,14 @@ If a receiver detects the CloudEvents media type, but with an event format that
 it cannot handle, for instance `application/cloudevents+avro`, it MAY still
 treat the event as binary and forward it to another party as-is.
 
+When the `Content-Type` header is not prefixed with the CloudEvents media type,
+being able to distinguish between a _binary_ CloudEvent and a non-CloudEvent
```

Comment:
So maybe the word "distinguish", at least to me, is unclear. With distinguish do you mean "start the parsing, even of a non valid event" or "start the parsing of a valid event?".

If I understood the goal here, you want to give an hint to the spec implementer on how to implement the parsing process of the HTTP envelope. If that's the case, I think it would be useful to specify that if it has the specversion, then it's an event, potentially non valid

Reply:
See comment below, from a spec perspective, if an incoming CE doesn't adhere to all of the MUSTs in the spec then it is not a valid CE. What a receiver chooses to do with it at that point is an impl choice - the spec is silent on that because we purposely decided not to get into the processing model side of things (for the most part).

So, I guess "distinguish" can be interpreted as "whether someone should try to parse it as a CE" yes, but even then there's no guarantee that it'll pass all of the MUST checks. I've added a little of of rewording -  see what you think.

Reply:
I like the rewording, thanks!